src/sql/base/browser/ui/table/tableDataView.ts

```diff
@@ -41,20 +41,26 @@ function defaultSort<T>(args: Slick.OnSortEventArgs<T>, data: Array<T>): Array<T
 
 export class TableDataView<T extends Slick.SlickData> implements IDisposableDataProvider<T> {
 	private _data: Array<T>;
+	private _allData: Array<T>;
```

Comment:
what's the difference between `allData` and `data`? Could you add a comment clarifying, or maybe rename `data` to something like `filteredData`?